core/tests/ts-integration/tests/api/web3.test.ts

```diff
@@ -183,7 +183,7 @@ describe('web3 API compatibility tests', () => {
         ['net_peerCount', [], '0x0'],
         ['net_listening', [], false],
         ['web3_clientVersion', [], 'zkSync/v2.0'],
-        ['eth_protocolVersion', [], 'zks/1'],
+        ['eth_protocolVersion', [], 'zks/0.28.0'],
```

Comment:
This will require frequent updates, right? Maybe, it makes sense to extract to a separate test and testing matching a regex like `^zks\/0\.\d+\.\d+$`?